src/ejabberd_ctl.erl

```diff
@@ -899,3 +453,30 @@ get_dist_proto() ->
         {ok, [Proto]} -> Proto;
         _ -> "inet_tcp"
     end.
+
+%%-----------------------------
+%% Cluster management commands
+%%-----------------------------
+
+join_cluster(NodeString) ->
+    handle_cluster_operation(join_cluster, [NodeString]).
+
+leave_cluster() ->
+    handle_cluster_operation(leave_cluster, []).
+
+remove_from_cluster(NodeString) ->
+    handle_cluster_operation(remove_from_cluster, [NodeString]).
+
+handle_cluster_operation(Operation, Args) ->
+    case apply(mongoose_server_api, Operation, Args) of
+        {ok, Result} ->
+            ?PRINT("~s\n", [Result]),
+            ?STATUS_SUCCESS;
+        {_, Result} ->
+            ?PRINT("Error: \"~s\"\n", [Result]),
+            ?STATUS_ERROR
+    end.
+
+cluster_command_without_arg() ->
```

Comment:
Actually this is also called when there is more than one argument, like
`$ _build/mim1/rel/mongooseim/bin/mongooseimctl join_cluster 1 2`, so maybe rename this function, e.g. `cluster_command_usage`